poutyne/framework/experiment.py

```diff
@@ -253,11 +396,53 @@ def train(self,
               batches_per_step=1,
               seed=42):
         # pylint: disable=too-many-locals
-        if seed is not None:
-            # Make training deterministic.
-            random.seed(seed)
-            np.random.seed(seed)
-            torch.manual_seed(seed)
+        """
+        Trains or finetunes the attribute model on a dataset using a generator. If a previous training already occured
+        and lasted a total of `n_previous` epochs, then the model's weights will be set to the best previous checkpoint
```

Comment:
That's not true. The model's weights will be set to the final weights after training.

Reply:
Isn't the call to _init_model_restoring_callbacks() done with initial_epoch > 1, meaning the model weights are restored to the best checkpoint's ones ? Or is there a second weight loading procedure after this one ?

Reply:
See _load_epoch_state.

Reply:
Okay, fixed by d461a875bc0b2548b0935ee1a6a93b940e78cfd1